Sources/JWT/Claims/ExpirationTimeClaim.swift

```diff
@@ -1,14 +1,14 @@
 import Foundation
 import Node
 
-public struct ExpirationTimeClaim: SecondsWithLeewayClaim {
+public struct ExpirationTimeClaim: TimeBasedClaim {
     public static var name = "exp"
 
-    public let leeway: Seconds
-    public let value: Seconds
+    let createTimestamp: () -> Seconds
+    let leeway: Seconds
 
-    public init(_ value: Seconds, leeway: Seconds = 0) {
-        self.value = value
+    public init(createTimestamp: @escaping () -> Seconds, leeway: Seconds) {
```

Comment:
I think the default value of `0` for leeway should stay. And the createTimestamp would be nicer for the caller with `@autoclosure`. (I won't copy this to nbf but obviously I mean for both places.)